control_plane/attachment_service/src/http.rs

```diff
@@ -402,6 +415,12 @@ pub fn make_router(
             request_span(r, handle_attach_hook)
         })
         .post("/debug/v1/inspect", |r| request_span(r, handle_inspect))
+        .post("/debug/v1/tenant/:tenant_id/drop", |r| {
```

Comment:
maybe the endpoint name could contain "debug" in it to make sure it's really just a testing endpoint. `debug_drop` maybe? Same for the other endpoint.

Reply:
Not sure I understand: the path is prefixed with `/debug/` for that reason

Reply:
Oh right, good point. Case rested.